src/http/index.js

```diff
@@ -29,6 +29,24 @@ function hapiInfoToMultiaddr (info) {
   return toMultiaddr(uri)
 }
 
+async function serverCreator (serverAddrsArr, createServerFunc, hapiInfoToMultiaddr, ipfs) {
```

Comment:
Please can we rename `serverAddrsArr` to just `serverAddrs` and then add a check as the first line of this function to convert it to an array if it is not already:

`serverAddrs = Array.isArray(serverAddrs) ? serverAddrs : [serverAddrs]`